src/options.ts

```diff
@@ -49,6 +50,10 @@ export class GitHubOptions implements Options {
     return core.getInput('branch')
   }
 
+  get force(): boolean {
+    return core.getInput('force')
```

Comment:
```suggestion
    return core.geBooleantInput('force')
```
To ensure the correct value type

Reply:
@fjogeleit done!